instrumentation/servlet/servlet-3.0/javaagent-tests/src/test/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/mapping/TomcatServlet3FilterMappingTest.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.servlet.v3_0.mapping;
+
+import java.io.IOException;
+import java.util.UUID;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.catalina.Context;
+import org.apache.tomcat.util.descriptor.web.FilterDef;
+import org.apache.tomcat.util.descriptor.web.FilterMap;
+
+abstract class TomcatServlet3FilterMappingTest extends TomcatServlet3MappingTest {
+  @SuppressWarnings("ClassNewInstance")
+  public void addFilter(Context servletContext, String path, Class<? extends Filter> filter) {
+    String name = UUID.randomUUID().toString();
+    FilterDef filterDef = new FilterDef();
+    try {
+      filterDef.setFilter(filter.newInstance());
+    } catch (Exception e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
I think you could add `throws Exception` to the method and just let the exception propagate